### PR TITLE
Made members page dynamic

### PR DESCRIPTION
**Pull request by mojcas**
Replaced static contents of members page with dynamic

### CODE REVIEW

File: web/templates/pages/members.html [hunk 1]
```diff
@@ -10,39 +10,22 @@
...
+	{% endif %}
+	{% if member.activity_status == '03' %}
+		<li><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Lazy cat)<li>
+	{% endif %}	
+	{% if member.activity_status == '04' %}
+		<li><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Sleepy cat)<li>
+	{% endif %}	
+	{% endfor %}
+	</ul>			
+	</div>
```
Comment (goranche) [inline web/templates/pages/members.html]: I think there might be a closing div tag missing... if you indent the content correctly, it might be easier to spot...

Comment (goranche) [inline web/templates/pages/members.html]: also, the header tag doesn't seem to be closed...

File: web/templates/pages/members.html [hunk 2]
```diff
@@ -10,39 +10,22 @@
...
+	{% endif %}	
+	{% if member.activity_status == '04' %}
+		<li><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Sleepy cat)<li>
+	{% endif %}	
+	{% endfor %}
+	</ul>			
+	</div>		
 
-{% endblock content%}
+{% endblock content%}
```
Comment (goranche) [inline web/templates/pages/members.html]: missing newline at end of file

File: api/models.py [hunk 3]
```diff
@@ -5,10 +5,10 @@
 class UserProfile(models.Model):
 
 	ACTIVITY_LEVEL = (
-			("SC", "Sleepy_Cat"),
-			("LC", "Lazy_Cat"),
-			("AC", "Active_Cat"),
-			("MC", "Mentor_Cat"),
+			("04", "Sleepy_Cat"),
```
Comment (goranche) [inline api/models.py]: why the changes to the model?
these changes would require a change in migrations, but there doesn't seem to be one.

I recommend these changes to be reverted

Comment (mojcas) [inline api/models.py]: These changes make it much easier to sort members by group on the page. It is not such a big change, I think. If noone strongly objects, I suggest we change these values in the model. If can not change these, then will need to figure out how to sort better on the member page.

Comment (ialja) [inline api/models.py]: Your template code doesn't change significantly if you use text codes instead of number codes. I'd also suggest leaving the model as it is for now.

Comment (mojcas) [inline api/models.py]: Sort by SC, LC, AC, MC is much more difficult, than 01, 02, 03, 04.. Why we must use these codes?

Comment (mojcas) [inline api/models.py]: ooh .. sure, I get it now.. okay, I'm on it :smile:

Comment (ialja) [inline api/models.py]: You're not using the sorting in the template - you check for the activity level anyway with the if statements (by the way, might be worth using [elif](https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#if)).

File: web/views.py [hunk 4]
```diff
@@ -13,7 +14,8 @@ def about(request):
 	return render(request, 'pages/about.html',{})
 
 def members(request):
-	return render(request, 'pages/members.html',{})
+	members_list = UserProfile.objects.filter(active=True).order_by('activity_status')
```
Comment (ialja) [inline web/views.py]: Suggestion: instead of activity_status ordering (which you're not using), you could sort alphabetically by first name :)

Comment (mojcas) [inline web/views.py]: There is no first name in the model, only fullname..

Comment (ialja) [inline web/views.py]: So use that :wink:

Comment (goranche) [inline web/views.py]: yes, and it's widely known one cannot order by fullname... :smile:

Comment (mojcas) [inline web/views.py]: I was trying to group members by active_status only didn't think it'd be so difficult. It does not really matter, so changed it. Happy now? :wink:

**Comment by goranche** [comment]
> Happy now?

I wouldn't go as far as saying that... I was quite happy before as well :stuck_out_tongue_winking_eye: 

you know what might be a good thing to add (as long as you already have the if structure), maybe add different classes to the \<li> tags for different classes of members...
it will make it easier to create design cues to differentiate the members... :smile:

**Comment by mojcas** [comment]
add different classes to the <li> tags for different classes of members... How do I do that?

**Comment by goranche** [comment]
@mojcas do you know how classes are added to HTML tags? if not, try searching for "HTML tag class"
(and of course, if you still have problems, we'll help :wink: )

**Comment by mojcas** [comment]
I mean, should I create new classes, or what classes should I use? I'm not so good on design, etc. :wink:

**Comment by goranche** [comment]
you can define your own ones, maybe copy them from the model (but use lower case), just so someone can address them from the CSS later on

**Comment by mojcas** [comment]
@goranche classes added.. but they don't seem to do much.. Still happy? :smile_cat:
What else? Now we wait for other peers review?
 In the meanwhile, caan I create another branch, to continue? :confused:

**Comment by goranche** [comment]
> to continue

depends on what "to continue" means to you... if it depends on the code in this PR, then I don't recommend it, because what if this code isn't merged into the repository, what if you need to change it some more? but in general, the answer is yes, you could do that
if you mean work on something unrelated, sure

**Comment by mojcas** [comment]
unrelated yes, the navigation .. #75 Navigation bar "folds" at certain widths :wink:

**Comment by mojcas** [comment]
It is *mostly* unrelated, just one wee bit of a problem.. Both pull requests #131 and #130 updating static/css/style.css .. Can this be merged? Thanks!

**Comment by goranche** [comment]
not sure what you mean by "can this be merged"...

if nothing else, when on of the PRs is accepted, you'll have to make sure the other PR will have the same changes applied on it as well (sync with upstream and master)...
this will make sure the changes are merged, yes

**Comment by fbbunny** [comment]
Tested this, works good with me; so as far as I am concerned our next step is to make a new issue: tasks.html. So instead of going to the members github account, you would be redirected to a page inside the webpage, which would show which tasks or projects a clicked member is assigned to.

**Comment by anuschka** [comment]
The profile_picture does not display for each active member (I added a picture for each user in the DB).

**Comment by mojcas** [comment]
@anuschka can you check right click on the picture and "Copy Image Location"? Also, when you add a picture in the DB, how you add it? Try like this, it works for me..
userprofile

**Comment by anuschka** [comment]
@mojcas I did exactly like you sad and created the exact user like you did and still no picture. I select a picture from my local drive and add ti to the profile picture in the admin.

**Comment by mojcas** [comment]
@anuschka can you check right click on the picture and "Copy Image Location"?

**Comment by anuschka** [comment]
@mojcas : Your PR is working great and the pictures are showing perfectly :+1:
